lib/formatters.js

```diff
@@ -109,12 +113,12 @@ var formatOutputUInt = function (value) {
 
 /// @returns input bytes formatted to real
 var formatOutputReal = function (value) {
-    return formatOutputInt(value).dividedBy(new BigNumber(2).pow(128)); 
+    return formatOutputInt(value).div(new BigNumber(2).toRed(BigNumber.red('k256')).redPow(new BigNumber(128)).fromRed()); 
```

Comment:
also see this issue on a brief explention of reduction context https://github.com/indutny/bn.js/issues/40